playbackSpeed/src/main/kotlin/voice/playbackSpeed/PlaybackSpeedComponent.kt

```diff
@@ -0,0 +1,122 @@
+package voice.playbackSpeed
+
+import SpeedRoulette
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Chip
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropUp
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import voice.common.compose.VoiceTheme
+
+@Composable
+fun PlaybackSpeedComponent(
```

Comment:
This component is doing quite a lot. It holds business logic but also is responsible for UI elements.
I think it makes sense to create a pure logic class that handles the selection and value creation logic and let the composable itself only render the final state.
That would also make this testable and prevent bugs like you have when you quickly fling this left and right

<img width="406" alt="image" src="https://user-images.githubusercontent.com/743462/191273060-728d93ce-94bc-4bef-952e-d67591a62c78.png">